plugins/guests/fedora/plugin.rb

```diff
@@ -21,10 +21,21 @@ class Plugin < Vagrant.plugin("2")
         Cap::ConfigureNetworks
       end
 
+      guest_capability("fedora", "flavor") do
+        require_relative "cap/flavor"
+        Cap::Flavor
+      end
+
       guest_capability("fedora", "network_scripts_dir") do
         require_relative "cap/network_scripts_dir"
         Cap::NetworkScriptsDir
       end
+
+      guest_capability("fedora", "nfs_client_install") do
+       	require_relative "cap/nfs_client"
+       	Cap::NFSClient
```

Comment:
The spacing is a little off here - I think this might be a tab instead of two spaces on accident. Could you please fix that up?